css/style.css

```diff
@@ -1,10 +1,142 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
 
-body {
+ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+header {
+  position: fixed;
+  top: 0;
+  min-height: 75px;
+  padding: 0px 50px;
+  display: flex;
+  justify-content: flex;
+  align-items: center;
+  background-color: #ffffff;
+ 
+}
+@media (max-width: 500px) {
+  header {
+    flex-wrap: wrap;
+  }
+}
+
+  
+.logo {
+    margin-top: 15px;
+    width: 10%;
+    position: relative;
+  }
+
+
+.logo > img {
+  width: 100%;
+  height: 3.375rem;
+  max-width: 200px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.nav-bar{
+  background-color: #ffffff;
+  height: 3.375rem;
+  width: 70rem;
```

Comment:
Your nav-bar is too wide here, it's breaking out of the screen. This is because of a couple of things:

- Your `<header>` needs a width of 100% so it's always the same width as the screen.
- The `nav-bar` section is fixed at 70rem and doesn't take into account the fact that the screen may be smaller than 70rem (then the nav items get cut off) or larger.

A good fix could be to remove the width on the nav-bar, and set the header to `width: 100%`. Once you've done that, can you think which flexbox rule you could use to ensure the logo always stays on the left, and the nav-bar always stay on the right?